Resources/Discover Distributions/qml/LDbernoulli.qml

```diff
@@ -0,0 +1,161 @@
+//
+// Copyright (C) 2013-2018 University of Amsterdam
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with this program.  If not, see
+// <http://www.gnu.org/licenses/>.
+//
+import QtQuick 2.8
+import QtQuick.Layouts 1.3
+import JASP.Controls 1.0
+import JASP.Widgets 1.0
+
+Form {
+  id: form
+
+  Section
+  {
+    expanded: true
+    title: qsTr("Show Distribution")
+    Group
+    {
+        title: qsTr("Parameter")
+        Layout.columnSpan: 2
+        DoubleField{ name: "prob"; label: qsTr("p"); id: prob; min: 0; max: 1; defaultValue: 0.5 }
+    }
+    
+    Group
```

Comment:
This is a great suggestion, I must admit the duplication of parts of qml caused me more than one moment of desperation; I will check that out! But could I maybe do this later given that it’s not completely necessary to have this before we start the testing phase? It would give me more time right now to focus on the issues that potentially deter the user experience :D

Reply:
> But could I maybe do this later given that it’s not completely necessary to have this before we start the testing phase?

Fine with me. @TimKDJ ?

Reply:
Yeah we can tidy up later. Would be good to have this in ASAP